docs/source/references/api_spec.md

```diff
@@ -146,28 +152,34 @@ continues to stream blocks as they are added to the blockchain.
 
 **Expects**:
 There are 2 options:
+
 - Expects: a JSON body `"Value"`. Returns: configuration value as JSON.
-- Expects: a JSON body that specifies the field (see example below). Returns: documentation for a specific field (as JSON string) or `null`.
+- Expects: a JSON body that specifies the field (see example below). Returns: documentation for a specific field (as
+  JSON string) or `null`.
 
-Note that if the requested field has more fields inside of it, then all the documentation for its inner members is returned as well.
+Note that if the requested field has more fields inside of it, then all the documentation for its inner members is
+returned as well.
 Here is an example for getting a field `a.b.c`:
+
 ```json
 {
-    "Docs": ["a", "b", "c"]
+  "Docs": ["a", "b", "c"]
 }
 ```
 
 **Examples**:
 To get the top-level configuration docs for [`Torii`] and all the fields within it:
+
 ```bash
 curl -X GET -H 'content-type: application/json' http://127.0.0.1:8080/configuration -d '{"Docs" : ["torii"]} ' -i
 ```
 
 **Responses**:
+
 - 200 OK: Field was found and either doc or value is returned in json body.
 - 404 Not Found: Field wasn't found
 
-### Configuration
+### Update Configuration
```

Comment:
Please revert. 

This is the configuration endpoint. 

Also, you don't update the configuration of something, you *configure* something. Configuration means also the process of configuring something.